preview/test/widget.spec.ts

```diff
@@ -52,11 +52,14 @@ test.describe('Preview widget', () => {
         await expect(page).toHaveScreenshot('widget-expanded.png');
     });
 
-    test('should not display the experiment if not specified', async ({page}) => {
+    test('should display the none audience for the slot default content preview', async ({page}) => {
```

Comment:
'Not display' is being used to express that nothing will be displayed, it is not the case here because we display the 'None' label in the audience (content out of any audience). What about:
`should not specify the audience when previewing the default content`?

Reply:
How about this?

> should not indicate any specific audience when previewing the default content